main.go

```diff
@@ -78,6 +79,12 @@ func main() {
 			Usage:  "use TLS and verify the remote",
 			EnvVar: "DOCKER_TLS_VERIFY",
 		},
+		cli.IntFlag{
+			Name:	"timeout",
+			Usage:	"timeout before container is forcefully stopped",
```

Comment:
I suggest saying "timeout **in seconds** before container is forcefully stopped", to be more clear